### PR TITLE
Changes needed for conda-build 2.0

### PR DESCRIPTION
**Pull request by jjhelmus**
Partial fix for #28

### CODE REVIEW

**Comment by jakirkham** [comment]
So, re-rendering this is a bit dicey as we have `conda-forge-build-setup` requiring itself to build. If it doesn't work, we can't fix it easily. Thus it could just get totally blocked. The only real way to fix it would provide an option so this feedstock could bootstrap itself. This could also provide an option for a temporary feedstock override if needed.

Put together PR ( https://github.com/conda-forge/conda-smithy/pull/285 ) to address this. See PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/30 ) for how this would work. Had some questions for @pelson, but have not heard back yet.

**Comment by jjhelmus** [comment]
Ahh, yes good point.  Would it make sense to revert 8475a84 so that this feedstock does not bootstrap itself?

**Comment by jakirkham** [comment]
Yeah, let's do that for now. We should figure out at some point how to re-render this feedstock safely.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package:
   name: conda-forge-build-setup
-  version: 3.2.0
+  version: 3.3.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: This now needs to be `3.4.0`.

**Comment by jjhelmus** [comment]
Rebased this PR to resolve the merge conflict.

**Comment by jakirkham** [comment]
LGTM. Care to add yourself as a maintainer? 😉

File: recipe/run_conda_forge_build_setup_win.bat [hunk 2]
```diff
@@ -11,8 +11,5 @@ conda config --set add_pip_as_python_dependency false
 conda update -n root --yes --quiet conda
 conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 
-:: KLUDGE to work around changes in conda-build 2.0.0
-conda install -n root --yes --quiet conda-build=1
```
Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win.bat:15]: So I think we still need these kludgy pinnings for now as there isn't a way to re-render with the upload script. Going to add them back in.

Comment (jakirkham) [reply recipe/run_conda_forge_build_setup_win.bat:15]: Please see PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/37 ).

**Pull request merged**